babel.config.js

```diff
@@ -1,10 +1,19 @@
 module.exports = {
-    plugins: ["const-enum", "@babel/transform-typescript"],
+    plugins: [
+        "const-enum",
```

Comment:
I don't think we need this anymore? Not really relevant to emotion so it doesn't need to be changed here but worth being aware of.